spec/features/consumer/caching/darkswarm_caching_spec.rb

```diff
@@ -45,6 +45,9 @@
         expect(page).to have_content property.presentation
       end
 
+      # Update rows which should also update the timestamp.
+      # The timestamp represents seconds, so waiting one second is enough.
+      sleep 1
```

Comment:
This is already included in another PR, right? Does it need to be here?

Reply:
I just wanted the specs to pass. :-) I will rebase this and the commit will just disappear...